src/tasks/AotCompilerTask/MonoAOTCompiler.cs

```diff
@@ -400,6 +421,41 @@ public override bool Execute()
         }
     }
 
+    private bool ProcessNettrace(string netTraceFile)
+    {
+        var outputMibcPath = Path.Combine(OutputDir, Path.ChangeExtension(Path.GetFileName(netTraceFile), ".mibc"));
+
+        if (_cache!.Enabled && File.Exists(outputMibcPath))
+        {
+            string hash = Utils.ComputeHash(netTraceFile);
```

Comment:
You want to add the hash of the source file to the cache, even if the output file doesn't exist yet.

Reply:
Ah, right that makes sense so we can skip after first instance of the generated mibc file (instead of the second)